terraform/modules/fargate-task-definition/outputs.tf

```
@@ -21,3 +21,8 @@ output "container_name" {
   description = "Name of the container"
   value       = var.container_name
 }
+
+output "log_group" {
+  description = "Name of the log group for OS"
+  value       = aws_cloudwatch_log_group.lamda_logs.name
```

```suggestion
  description = "Log group for OS"
  value       = {
      name = aws_cloudwatch_log_group.lamda_logs.name
      arn = aws_cloudwatch_log_group.lamda_logs.arn
```
That will need `terraform fmt` applying